packages/rum/src/domain/record/record.spec.ts

```diff
@@ -334,20 +335,26 @@ describe('record', () => {
       expect(inputRecords.length).toBe(1)
     })
 
-    it('should record the scroll event inside a shadow root', () => {
-      const div = appendElement('<div></div>', createShadow()) as HTMLDivElement
+    fit('should record the scroll event inside a shadow root', () => {
+      const div = appendElement('<div unique-selector="enabled"></div>', createShadow()) as HTMLDivElement
       startRecording()
       expect(getEmittedRecords().length).toBe(recordsPerFullSnapshot())
 
       div.dispatchEvent(createNewEvent('scroll', { target: div, composed: false }))
 
       recordApi.flushMutations()
-      const innerMutationData = getLastIncrementalSnapshotData<BrowserMutationData>(
+      const innerMutationData = getLastIncrementalSnapshotData<ScrollData>(
         getEmittedRecords(),
         IncrementalSource.Scroll
       )
 
-      expect(innerMutationData).toBeDefined()
+      const fs = findFullSnapshot({ records: getEmittedRecords() })!
+      const scrollableNode = findElement(
+        fs.data.node,
+        (node) => { return node.attributes['unique-selector'] === "enabled" }
+      )!
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would it make sense to also check that there is no other scroll records emitted for other elements?